app/main.go

```diff
@@ -133,6 +140,12 @@ func main() {
 		openAIBot,
 	}
 
+	if opts.SpamFilter.Enabled {
+		log.Printf("[INFO] spam filter enabled")
```

Comment:
Maybe worth adding API either here or under the debug log entry as we will forget what is it and how it works eventually. Would be great to see it from logs and not only in the code.